File: src/main/java/kr/syeyoung/dungeonsguide/mod/whosonline/api/messages/client/C01IsOnline.java
package kr.syeyoung.dungeonsguide.mod.whosonline.api.messages.client;

import lombok.Data;

@Data
public class C01IsOnline {
    public final String t = "/is_online";

    /**
     * The uuid to check
     */
    public final OBJ c;

    @Data
    public static class OBJ {
        public final String uuid;
        public final String nonce;
    }

}

File: src/main/java/kr/syeyoung/dungeonsguide/whosonline/api/messages/client/C01IsOnline.kt
package kr.syeyoung.dungeonsguide.whosonline.api.messages.client

import lombok.AllArgsConstructor
import lombok.Data

@Data
@AllArgsConstructor
class C01IsOnline(val c: OBJ) {
    val t = "/is_online"


    data class OBJ(val uuid: String?, val nonce: String?)

}